docs/hacking.rst

```diff
@@ -2,273 +2,256 @@
 Hacking
 =======
 
-L20n library is a combination of three layers of APIs built on top of each other.
-The intention is for the user to be able to use just the first or first and the second
-layer separately.
+The L20n library is built as a combination of three layers of APIs stacked on 
+top of each other.  This modularization allows l20n.js to support many 
+different environments and platforms.
 
-Each layer has different external dependencies and is intended to tie better into the
-host and client environments.
+:ECMA 402 Intl
+    The low-level API providing L20n's primitives and building-blocks.
```

Comment:
Similarily, I think we should call this something like "L20n builds on top of JavaScript Intl APIs. We also work with ECMA to enhance these APIs where there's overlap between the needs of L20n and other libraries needing access to internationalized data."